drupal/sites/all/features/user_profiles/user_profiles.features.content.inc

```diff
@@ -492,9 +492,9 @@ Zimbabwe|Zimbabwe',
       'default_value_php' => NULL,
       'label' => 'Your opinions about this project',
       'weight' => '4',
-      'description' => 'Tell us your thoughts about Einstein@Home
+      'description' => 'Tell us your thoughts about ' . variable_get('site_name', '') . '
```

Comment:
Same comment as for #1834. In translations the order of words changes based on the language grammar, please use a substitution instead of a concatenation.

Also it seems that these strings do not get translated right now but that might be fixed by #1834. It's shown on account/profile/edit.